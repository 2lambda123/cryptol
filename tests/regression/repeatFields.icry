let z = { a = 0x08,  a = "asdf" }

let g (x: { b : [8], b : String 4 }) = x.b

let h { c = v, c = u } = zero#v == u
