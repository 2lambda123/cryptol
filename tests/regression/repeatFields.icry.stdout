Loading module Cryptol

Parse error at repeatFields.icry:1:22--1:23
  Record has repeated field: a

Parse error at repeatFields.icry:3:22--3:23
  Record has repeated field: b

Parse error at repeatFields.icry:5:16--5:17
  Record has repeated field: c
